lib/ex_unit/examples/difference.exs

```diff
@@ -25,6 +25,12 @@ defmodule Difference do
     assert string1 == string2
   end
 
+  test "whitespace" do
+    list1 = [%{a: "abc "}, %{a: "def"}, %{c: "gh"}]
+    list2 = [%{a: "abc"}, %{a: " def"}, %{c: "ghi"}]
```

Comment:
We can use `"hi"` for last string here, so we get `:diff_delete` color for previous line.